djangosaml2/backends.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, **kwargs):
         self.kwargs = kwargs
 
     def authenticate(self, session_info=None, attribute_mapping=None,
```

Comment:
We can't remove the tenant from here as even though we don't use it here, the BelegatingBackend wants `authenticate ` method's signature to have a tenent.